client/src/fabric/fabric-exports.ts

```diff
@@ -7,7 +7,7 @@ export interface IMemory {
     buffer: ArrayBuffer
 }
 
-export enum Direction {
+export enum FabricState {
```

Comment:
can you name this something more descriptive than "FabricState"?

Reply:
no idea what would be better.

Reply:
actually you can completely ignore the directions thing until we revive Galapagotchi.  all tensegrity fabrics currently stay in rest-state forever.  soon this might become FabricShape or something, because it will be a "recording" a particular set of ideal length values between which can be interpolated, for running behavior.

Reply:
ok